app/src/services/search/local/local-search.ts

```diff
@@ -0,0 +1,78 @@
+import mingo from "mingo";
+import assert from "assert";
+
+export type ConnectionsSearchMatch = {
```

Comment:
Just curious, why types instead of interface?

Reply:
I figured since we're not extending the type we could use types instead.

Reply:
On second thought, I think interfaces are a better fit here.